libs2eplugins/src/s2e/Plugins/Core/BaseInstructions.h

```diff
@@ -74,6 +74,7 @@ class BaseInstructions : public Plugin, public IPluginInvoker {
     void onTranslateBlockStart(ExecutionSignal *signal, S2EExecutionState *state, TranslationBlock *tb, uint64_t pc);
 
     void forkCount(S2EExecutionState *state);
+    void forkSimple(S2EExecutionState *state);
```

Comment:
Could you perhaps instead reuse `forkCount` and use your fork api when the variable name is null? This would minimize the amount of changes.